app/scripts/views/sign_up.js

```diff
@@ -178,6 +179,11 @@ define(function (require, exports, module) {
         showSyncSuggestion: this.isSyncSuggestionEnabled()
       };
 
+      if (device._hasWebChannelSupport ()) {
+        context.escapedSyncSuggestionUrl = encodeURI('https://accounts.firefox.com/signup?service=sync&context=fx_desktop_v3&entrypoint=uitour');
```

Comment:
`https://accounts.firefox.com/` value needs to picked up from the config or be relative to the application